src/components/ui-shell/_variables.scss

```diff
@@ -0,0 +1 @@
+$unit: 8px;
```

Comment:
Should we use `rem` to accommodate zooming?

Reply:
Up for anything! Whatever works best 😄